### ISSUE TITLE
Boltzmann/Data/Oracle.hs: No instance for (Semigroup Nat)

### ISSUE THREAD

**Issue by juhp** [Lysxia/boltzmann-samplers#2]
Not sure why this started in the Stackage Nightly build:

```
src/Boltzmann/Data/Oracle.hs:100:10: error:
    • No instance for (Semigroup Nat)
        arising from the superclasses of an instance declaration
    • In the instance declaration for ‘Monoid Nat’
    |
100 | instance Monoid Nat where
    |          ^^^^^^^^^^

src/Boltzmann/Data/Oracle.hs:268:41: error:
    • No instance for (Semigroup Nat) arising from a use of ‘<>’
    • In the expression: order <> order'
      In the expression: (order <> order', lCoef * lCoef')
      In an equation for ‘lMul’:
          lMul (order, lCoef) (order', lCoef')
            = (order <> order', lCoef * lCoef')
    |
268 | lMul (order, lCoef) (order', lCoef') = (order <> order', lCoef * lCoef')
    |                                         ^^^^^^^^^^^^^^^
```

**Comment by Lysxia** [comment]
Thanks! I added the missing instance in 816d4f6, released in 0.1.1.0.

@juhp  Should I just make a PR to stackage to reenable it?

**Comment by juhp** [comment]
I reverted thanks
